yt_dlp/cookies.py

```diff
@@ -347,6 +347,11 @@ def _process_chrome_cookie(decryptor, host_key, name, value, encrypted_value, pa
         if value is None:
             return is_encrypted, None
 
+    if has_expires == 1:
+        expires_utc = expires_utc
+    else:
+        expires_utc = None
```

Comment:
For `has_expires`, only `0` should signal false? Also the upper branch did nothing:
```suggestion
    if not has_expires:
        expires_utc = None
```

Reply:
maybe add a comment here too